spec/ics-003-connection-semantics/README.md

```diff
@@ -318,24 +330,27 @@ function connOpenTry(
     abortTransactionUnless(validateConnectionIdentifier(desiredIdentifier))
     abortTransactionUnless(consensusHeight < getCurrentHeight())
     expectedConsensusState = getConsensusState(consensusHeight)
-    expected = ConnectionEnd{INIT, desiredIdentifier, getCommitmentPrefix(), counterpartyClientIdentifier,
+    abortTransationUnless(
+      provedIdentifier === desiredIdentifier ||
```

Comment:
This is a bit confusing to me.

proved identifier is empty or comes from the init chain.

Desired Identifier comes from the receiving chain. But where? There doesn't seem to be a way to say "use whatever name they suggest", or "use whatever name they suggest if it starts with 'foo'" for example. 

Rather than comparing here, it would make more sense to have

`function desiredIdentifier(provedIdentifier) -> Identifier {}`

So the receiving module can make this decision and about if desired

Reply:
Right, that's the intended usage, it just doesn't need to happen "within" the IBC module, per se. I can add a note about it.

Reply:
Added a comment with your example.